assets/js/components/devices/import/ImportDevicesModal.jsx

```diff
@@ -112,7 +113,36 @@ const ImportDevicesModal = (props) => {
               <Title style={{width: '100%', textAlign: 'center'}}>Import Status</Title>
               {importStarted && !importComplete && <Spin indicator={antLoaderGrey} style={{ marginBottom: 20 }}/>}
               {importStarted && !importComplete && <Text style={{ textAlign: 'center' }}>Please wait while your import is being completed.</Text>}
-              {importComplete && <Text style={{ textAlign: 'center' }}>Your import is complete, please return to the device index page to view your new devices.</Text>}
+              {importComplete && (
+                <div>
+                  {
+                    import_status.successful_count && (
+                      <Text style={{ display: 'block', marginBottom: 12, textAlign: 'center' }}>
+                        Successfully imported {import_status.successful_count} device{(import_status.successful_count !== 1 && "s") || ""}, please return to the device index page to view your new devices.
+                      </Text>
+                    )
+                  }
+                  {
+                    import_status.failed_devices.length > 0 && (
+                      <Text style={{ display: 'block', marginBottom: 12, textAlign: 'center' }}>
+                        Errors Encountered:
+                      </Text>
+                    )
+                  }
+                  {
+                    import_status.failed_devices.length == 0 && (
```

Comment:
this will show even if all are successful

![image](https://user-images.githubusercontent.com/51131939/129288127-6dc50ce1-abda-49a0-8209-dea1b476a4e1.png)

Reply:
or maybe it's gotta be if count of successful is different than total attempted?